packages/manager/modules/manager-pci-common/src/components/modal/PciModal.component.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  OsdsButton,
+  OsdsModal,
+  OsdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_BUTTON_VARIANT, ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+
+type PciModalProps = {
+  children: React.ReactNode;
+  type: 'warning' | 'default';
+  title: string;
+  isPending: boolean;
+  isDisabled: boolean;
+  cancelText: string;
+  submitText: string;
```

Comment:
Maybe you can make cancelText & submitText optional too ?
with default values common_cancel & common_submit (something like this depending on available common translations)